powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/entity/HttpAuthenticationPublic.java

```diff
@@ -37,11 +38,15 @@ public static class Certificate {
         private boolean enabled;
         private boolean useCustomKeyStore;
         private String keyStoreLocation;
+        @ToString.Exclude
+        private String keyStoreContent;
```

Comment:
Do we really want to expose `keyStoreContent` to the REST API response? Why not follow the same pattern with a flag only such as `keyStorePasswordSet`? Moreover, I do not see changes in the PowerAuth-Admin. Is it intentional?

Reply:
Good point, I will add the flag.

As for PowerAuth admin, I do not think we need to add it now.